src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: Begin root cause analysis with the APM summary page
+tags:
+  - APM
+  - Triage
+  - Diagnostics
+metaDescription: Use our application monitoring tools to quickly triage and run diagnostics during an incident.
+---
+
+import apmApmCompleteSummaryPage from 'images/apm_screenshot-full_apm-complete-summary-page-.webp'
+
+import apmDistributedTracingintheApmSummaryPage from 'images/apm_screenshot-crop_distributed-tracing-in-the-apm-summary-page.webp'
+
+import apmWebTransactionsinAPM from 'images/apm_screenshot-crop_web-transactions-in-APM.webp'
+
+import apmApdexinApm from 'images/apm_screenshot-crop_apdex-in-apm.webp'
+
+import apmThroughputinApm from 'images/apm_screenshot-full_throughput-in-apm.webp'
+
+import apmErrorsinAPM from 'images/apm_screenshot-full_errors-in-APM.webp'
+
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
+
+import apmLogsinApm from 'images/apm_screenshot-crop_logs-in-apm.webp'
+
+We all know what it's like to wake up in the morning to a ping from your leadership team telling you that something is going wrong and they need it fixed as soon as possible. This is where our <InlinePopover type="apm" /> tools come in. When you have limited time and increased urgency, you can use the APM summary page to to find failures in your applications and triage issues from one place.
+
+The APM summary page is an overview of the important metrics for your service. It's the first place you will go to investigate the source of a service degradation. On the APM summary page you can check the health of your service, understand any problems within a larger context, and take the necessary steps to resolve the issue. 
+
+<img
+    title="apm summary page"
+    alt="A screenshot depicting the APM summary page."
+    src={apmApmCompleteSummaryPage}
+/>
+
+This doc walks you through a _**fictional**_ example of how to use New Relic's APM tools to determine the root cause of service degradation. We're going to explore a situation where **you work for a fictional e-commerce company that sells shoes**. You are an engineering manager the checkout team.  Customers have been complaining they have to wait too long to checkout. 
+
+To investigate, open APM and select your `Checkout-service`.
+
+
+<Steps>
+<Step>
+
+## First, review these key metrics
+Is there a problem with your `Checkout-service`? Which systems are affected and how?
+
+<CollapserGroup>
+  <Collapser
+    id="apdex"
+    title="Apdex"
+  >
+
+<img
+    title="apdex in APM"
+    alt="A screenshot depicting a sample Apdex score in APM"
+    src={apmApdexinApm}
+/>
+The first place to look when investigating a breakdown in service is your Apdex score. Your Apdex score monitors overall customer satisfaction with your application. Your score looks for a combination of performance, like response time or throughput, and error rates. 
```

Comment:
I like that you explain the Apdex score. Is there a place we have in documentation that fully lists out everything about it that we can link them to as a read-more kind of thing? Just a thought